packages/now-cli/src/util/get-files.ts

```diff
@@ -415,24 +411,6 @@ export async function docker(
   return uniqueStrings(files);
 }
 
-/**
- * Get a list of all files inside the project folder
- *
- * @param {String} of the current working directory
- * @param {Object} output instance
- * @return {Array} of {String}s with the found files
- */
-export async function getAllProjectFiles(cwd: string, { debug }: Output) {
-  // We need a slash at the end to remove it later on from the matched files
-  const current = join(resolve(cwd), '/');
-  debug(`Searching files inside of ${current}`);
-
-  const list = await glob('**', { cwd: current, absolute: true, nodir: true });
-
-  // We need to replace \ with / for windows
```

Comment:
Does the new function work with Windows? I assume so but it’s good to test it out first to confirm.

Reply:
It does. The `relative()` helper function that is used in `dev/server.ts` does this same replacement.